sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Release History
 
 ## 12.7.0-beta.1 (Unreleased)
+
+### Features Added
+
```

Comment:
we don't follow this format in storage changelogs. would you mind removing ### sub-paragraphs? here and in other changelogs.